kbase-extension/static/kbase/js/common/jobCommChannel.js

```diff
@@ -320,6 +329,13 @@ define([
         handleCommMessages(msg) {
             const msgType = msg.content.data.msg_type;
             const msgData = msg.content.data.content;
+            const msgDataType = Utils.objectToString(msgData);
+
+            // message data should be an object
+            if (msgDataType !== 'Object') {
```

Comment:
Are there cases where this can fail?

Reply:
no, the BE always returns an object -- hurrah!